dev/README_AIRFLOW3_DEV.md

```diff
@@ -58,29 +58,33 @@ PRs should target `main` branch.
 Make sure your code is only about Providers or Helm chart.
 Avoid mixing core changes into the same PR
 
+> [!NOTE]
+> Please note that providers have been relocated from `airflow/providers` to `providers/src/airflow/providers`.
+
 ## Developing for Airflow 3 and 2.10.x / 2.11.x
 
 If the PR is relevant for both Airflow 3 and 2, it should target `main` branch.
 
-Note: The mental model of Airflow 2.11 is bridge release for Airflow 3.
-As a result, Airflow 2.11 is not planned to introduce new features other than ones relevant as bridge release for Airflow 3.
-That said, we recognize that there may be exceptions.
-If you believe a specific feature is a must-have for Airflow 2.11, you will need to raise this as discussion thread on the mailing list.
-Points to address to make your case:
-
-1. You must clarify what is the urgency (i.e., why it can't wait for Airflow 3).
-2. You need be willing to deliver the feature for both main branch and Airflow 2.11 branch.
-3. You must be willing to provide support future bug fixes as needed.
-
-Points to consider on how PMC members evaluate the request of exception:
-
-1. Feature impact - Is it really urgent? How many are affected?
-2. Workarounds - Are there any ?
-3. Scope of change - Both in code lines / number of files and components changed.
-4. Centrality - Is the feature at the heart of Airflow (scheduler, dag parser) or peripheral.
-5. Identity of the requester - Is the request from/supported by a member of the community?
-6. Similar previous cases approved.
-7. Other considerations that may raise by PMC members depending on the case.
+> [!IMPORTANT]
+> The mental model of Airflow 2.11 is bridge release for Airflow 3.
+> As a result, Airflow 2.11 is not planned to introduce new features other than ones relevant as bridge release for Airflow 3.
+> That said, we recognize that there may be exceptions.
+> If you believe a specific feature is a must-have for Airflow 2.11, you will need to raise this as discussion thread on the mailing list.
+> Points to address to make your case:
+>
+> 1. You must clarify what is the urgency (i.e., why it can't wait for Airflow 3).
+> 2. You need be willing to deliver the feature for both `main` branch and Airflow 2.11 branch.
+> 3. You must be willing to provide support future bug fixes as needed.
+>
+> Points to consider on how PMC members evaluate the request of exception:
+>
+> 1. Feature impact - Is it really urgent? How many are affected?
+> 2. Workarounds - Are there any ?
+> 3. Scope of change - Both in code lines / number of files and components changed.
+> 4. Centrality - Is the feature at the heart of Airflow (scheduler, dag parser) or peripheral.
+> 5. Identity of the requester - Is the request from/supported by a member of the community?
+> 6. Similar previous cases approved.
+> 7. Other considerations that may raise by PMC members depending on the case.
```

Comment:
```suggestion
> 1. You must clarify the urgency, specifically why it can't wait for Airflow 3.
> 2. You need to be willing to deliver the feature for both the `main` branch and the Airflow 2.11 branch.
> 3. You must be willing to provide support for future bug fixes as needed.
>
> Points to consider on how PMC members evaluate the request for exception:
>
> 1. Feature impact - Is it really urgent? How many are affected?
> 2. Workarounds - Are there any?
> 3. Scope of change - Both in code lines / number of files and components changed.
> 4. Centrality - Is the feature at the heart of Airflow (scheduler, dag parser) or peripheral.
> 5. Identity of the requester - Is the request from/supported by a member of the community?
> 6. Approved cases with similar details in the past.
> 7. Other considerations that may be raised by PMC members depending on the case.
```